### PR TITLE
fix: use new category_set key for review section

### PR DESCRIPTION
**Pull request by liruchen32**
# Issue
[報導者首頁「評論」區塊未顯示最新文章](https://app.asana.com/0/0/1206238664023561/f)

# Notice

# Dependency

### CODE REVIEW

File: controllers/news_v2.go [hunk 1]
```diff
@@ -302,7 +302,7 @@ func (nc *newsV2Controller) getIndexPageJobs() []job {
 			Name: news.ReviewsSection,
 			Type: typePost,
 			Query: news.NewQuery(
-				news.WithFilterCategoryIDs(news.Review.ID),
```
Comment (Aylie-Chou) [inline controllers/news_v2.go:305]: Question: is `Review` const been used in other place ? (If not, could we remove it)

**Review by Aylie-Chou — CHANGES_REQUESTED** [review]
Please have a look at comment ~

File: controllers/news_v2.go (hunk 1 above)
Comment (liruchen32) [reply controllers/news_v2.go:305]: fixed

**Review by Aylie-Chou — APPROVED** [review]
LGTM

**Pull request merged**